s2spy/preprocess.py

```diff
@@ -209,13 +209,16 @@ def __init__(  # noqa: PLR0913
         self._trend: dict
         self._is_fit = False
 
-    def fit(self, data: Union[xr.DataArray, xr.Dataset]) -> None:
+    def fit(self, data: Union[xr.DataArray, xr.Dataset], dropna=False) -> None:
         """Fit this Preprocessor to input data.
 
         Args:
             data: Input data for fitting.
+            dropna: If True, drop all NaN values from the data before preprocessing.
```

Comment:
```suggestion
            dropna: If True, drop all NaN values from the data along time axis before preprocessing.
```

Point out that this `dropna` works only for the time axis.